Design/ISIS_Imaging/ISIS_Imaging_general.md

```diff
@@ -109,72 +160,72 @@ Slices - referrs to a reconstructed sinogram. ![Reconstructed slice](https://git
 
 Attenuation - the result from performing a `-ln(image)`, which can be written as `numpy.negative(numpy.log(image))`. Depending on the dataset adding the `-log` to pre-processing can _significantly_ improve the reconstruction results and help remove noise!
 
-# Finalising name choice
+# 6. Finalising name choice
 
 - MantidImaging
 - ISIS_Imaging
 - Other suggestions?
 
-# Project setup
+# 7. Project setup
 
-## Continous Integration with Coverage
+## 7.1. Continous Integration with Coverage
 
 The package should have a continous integration flow on Pull Requests, which runs all of the CORE and GUI tests. This should also include coverage.
 
 After sphinx documentation is added, the contiguous integration could also check if it builds successfully.
 
-## Installation
+## 7.2. Installation
 
 We can either work towards the package being installable using `python setup.py install`, making it available system wide. The alternative solution is to use it from the folder and provide global access like [the example here](https://mantidproject.github.io/isis_imaging/user_guide/setting_up.html#global-access-to-the-package)
 
-## Testing
+## 7.3. Testing
 
 Testing will use `nose` to run the tests and `coverage` to compute the tests' coverage. The actual unit tests should use the built-in `unittest`, and some might have to use `numpy.testing` for asserting equality for `numpy.ndarrays`.
 
 A module to run the tests has been provided and is called `run_tests.py`, in the root of the repository. It takes as an argument the names of different packages, and then runs the tests specifically for them. The possible arguments are listed via `-h` flag, or at the top of the file in the source.
 
 There is currently no integration with Mantid's testing.
```

Comment:
Is there a reason why it should be integrated with Mantid testing? Seems completely separate to me. Any other views?